lib/galaxy/config/__init__.py

```diff
@@ -1103,10 +1104,36 @@ def _process_config(self, kwargs: Dict[str, Any]) -> None:
         self.proxy_session_map = self.dynamic_proxy_session_map
         self.manage_dynamic_proxy = self.dynamic_proxy_manage  # Set to false if being launched externally
 
-        # InteractiveTools propagator mapping file
-        self.interactivetools_map = self._in_root_dir(
+        # InteractiveTools propagator mapping
+        self.interactivetools_map = "sqlite:///" + self._in_root_dir(
             kwargs.get("interactivetools_map", self._in_data_dir("interactivetools_map.sqlite"))
         )
+        if self.interactivetools_map_sqlalchemy:
+            # ensure the database URL for the SQLAlchemy map does not match that of a Galaxy DB
+            urls = {
+                setting: parse_sqlalchemy_url(value)
+                for setting, value in (
+                    ("interactivetools_map_sqlalchemy", self.interactivetools_map_sqlalchemy),
+                    ("database_connection", self.database_connection),
+                    ("install_database_connection", self.install_database_connection),
+                )
+                if value is not None
+            }
```

Comment:
This is [causing workflow "Test" to fail](https://github.com/galaxyproject/galaxy/actions/runs/10043692668/job/27757024477?pr=18481).

```
____________ ERROR collecting tests/config/test_path_resolves_to.py ____________
ImportError while importing test module '/home/runner/work/galaxy/galaxy/galaxy root/packages/config/tests/config/test_path_resolves_to.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.12.4/x64/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/config/test_path_resolves_to.py:5: in <module>
    from galaxy.config import BaseAppConfiguration
galaxy/config/__init__.py:36: in <module>
    from sqlalchemy.engine import make_url as parse_sqlalchemy_url
E   ModuleNotFoundError: No module named 'sqlalchemy'
```

Any hint how to make the module available to the tests?